mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -210,6 +212,9 @@
 
     private static final String CDP_ALLOW_INTERNAL_REPOSITORY_FOR_UPGRADE = "CDP_ALLOW_INTERNAL_REPOSITORY_FOR_UPGRADE";
 
+    // See com.cloudera.thunderhead.service.common.entitlements.CdpEntitlements.CDP_CP_CUSTOM_DL_TEMPLATE
+    private static final String CDP_CP_CUSTOM_DL_TEMPLATE = "CDP_CM_ADMIN_CREDENTIALS";
```

Comment:
bit strange that the variable name is different from the value. would this mean `CDP_CM_ADMIN_CREDENTIALS` has the right to use/create custom templates?

Reply:
Yes. The name is indeed totally confusing. I am not sure what the originally intended name was supposed to be, but wanted to keep the thunderhead constant name here for "consistency". For reference, the check in DatalakeApi svc is made [here](https://github.infra.cloudera.com/thunderhead/thunderhead/blob/master/services/api/datalake/src/main/java/com/cloudera/thunderhead/service/datalakeapi/server/DatalakeApiImpl.java#L1095-L1107).

Reply:
Shall I rather rename the constant to match the actual entitlement name?

Reply:
I'm  fine with it, it just seemed interesting and could indicate some error. but @keyki should check this

Reply:
It's fine with me. If we would rename it to CDP_CM_ADMIN_CREDENTIALS probably it would be totally misleading what is it supposed to guard. But, indeed I'm not sure what this entitlement supposed to do.  Seems like a bad copy-paste error: https://github.infra.cloudera.com/thunderhead/thunderhead/blob/4cf3cae30759e47df352b2b41bc8b9c30b2c43b8/services/libs/common/src/main/java/com/cloudera/thunderhead/service/common/entitlements/CdpEntitlements.java#L510.

Reply:
Yeah. I fear it is too late to "rename" the entitlement as it might be already in use in several tenants. 😃